src/components/runtime-common.js

```diff
@@ -110,7 +110,7 @@ export const PeriodicCookieSetter = () => {
   return null
 }
 
-export const SaveFilesHelp = () => {
+export const SaveFilesHelp = (isGalaxyDisk = false) => {
```

Comment:
This wasn't part of my ticket but while testing my changes, I noticed that we are displaying a piece of message specific to Jupyter notebooks (from line#122 below) when Galaxy disk is to be deleted, and wanted to fix that as well.